tests/integration.rs

```diff
@@ -4258,7 +4257,7 @@ mod tests {
 
             let mut child = GuestCommand::new(&guest)
                 .args(&["--api-socket", &api_socket])
-                .args(&["--cpus", "boot=4"])
+                .args(&["--cpus", "boot=1"])
```

Comment:
Open an issue as we need to try and fix why it doesn't work with multiple vCPUs.

Reply:
Yes let me do that!

Reply:
https://github.com/cloud-hypervisor/cloud-hypervisor/issues/1176